.github/workflows/build-blog-only.yml

```
@@ -7,14 +7,17 @@ on:
     paths:
       - packages/docusaurus/**
 
+permissions:
+  contents: read
+
 jobs:
   build:
     name: Build Blog-only
     timeout-minutes: 30
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@b0e28b5ac45a892f91e7d036f8200cf5ed489415 # v3
```

```suggestion
      - uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846 # v3
```

Bad sha, != from others